src/InMemory/functions.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the zenstruck/foundry package.
+ *
+ * (c) Kevin Bond <kevinbond@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Zenstruck\Foundry\InMemory;
+
+use Zenstruck\Foundry\Configuration;
+
+/**
+ * Enable "in memory" repositories globally.
+ */
+function enable_in_memory(): void
+{
+    Configuration::instance()->enableInMemory();
```

Comment:
At first, I did something like:
```php
Configuration::instance()->persistence()->enableInMemory();
```

and the `$inMemory` boolean was stored in `PersistenceManager`. But then, all `InMemoryTest` did not worked when no persistence is activated, which is a non-sense :shrug: 

Then I decided to store this in the `Configuration` class